back/boxtribute_server/graph_ql/definitions/protected/types.graphql

```diff
@@ -447,6 +476,11 @@ input FilterProductInput {
   type: ProductTypeFilter = Custom
 }
 
+input FilterStandardProductInput {
```

Comment:
Change to only include baseID and return by default all newest standard products + all enabled standard products. (distinct)
If baseID is not passed return all newest standard products.